tools/epilogue.bt.fragment

```diff
@@ -0,0 +1,7 @@
+END {
+    system("rm $TMP_FILE");
+}
```

Comment:
We only need to delete the temp file if we use exec.  See the comment on `run.py`.

Reply:
See above